### PR TITLE
Add Client::setItemBorderColor

### PR DESCRIPTION
**Pull request by Abextm**
This will be required by #361

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/callback/Hooks.java [hunk 1]
```diff
@@ -72,6 +74,12 @@
 
 	public static void clientMainLoop(Client client, boolean arg1)
 	{
+		client.clearItemBorderColors();
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/callback/Hooks.java]: What is the point of this event? You can just create the subscription to gameTick or scheduled method in respective plugin

**Comment by Abextm** [comment]
AFAIK this runs every frame, not every tick.  When I set it to run every
tick it would lag a tick behind.

**Comment by deathbeam** [comment]
So you can do the processing what would be subscribed to the Reregister hook in the render method of the plugin instead, the `ReregisterItemBorders` hook is not really necessary at all.

**Comment by Abextm** [comment]
When I tried that nothing rendered at all, presumably because render is ran
after the client had rendered.

**Comment by deathbeam** [comment]
Then I guess renaming the event to something more generic should do. Like game loop or something.

**Pull request closed without merging**